core-plugin/src/com/google/gct/idea/debugger/ui/ExitDialog.java

```diff
@@ -18,9 +18,11 @@
 public class ExitDialog extends DialogWrapper {
```

Comment:
I'd like us to refactor this to once Appu's PR to refactor UsageTracker as a service is in.  Ideally services would be retrieved using their getInstance methods and we would mock them in tests using the new base test case.